Jenkinsfile

```diff
@@ -16,12 +16,16 @@ pipeline {
             steps {
                 // Delete old files
                 sh 'rm -rf .[^.] .??* *'
-                
```

Comment:
Looks good just a small nitpick: Split the style edits and the functional changes in separate commits.

Reply:
Yeah, normally I would but this was purely whitespace.

Reply:
Shouldn't matter really. They are logically two different commits. The history might seem confusing otherwise IMO or cherry picking might be complicated because of it.